src/humanize/i18n.py

```diff
@@ -7,11 +7,14 @@
 _TRANSLATIONS = {None: gettext_module.NullTranslations()}
 _CURRENT = local()
 
-try:
-    _DEFAULT_LOCALE_PATH = os.path.join(os.path.dirname(__file__), "locale")
-except AttributeError:
-    # in case that __file__ does not exist
-    _DEFAULT_LOCALE_PATH = None
+
+def get_default_locale_path():
```

Comment:
One last thought, should this be marked as an internal function or is it something we want to expose (and document via a docstring) via the API?

```suggestion
def _get_default_locale_path():
```

Reply:
Resolved marking as internal function.